exercises/calculator.cpp

```diff
@@ -62,6 +44,33 @@ void operations(float a, float b){
       break;
     }
   }
+}
+
+void operations(float a, float b){
+
+  int op;
+  float result;
+
+  while(true){
+
+    cout << endl << "Select the operation: " << endl << "1) SUM" << endl << "2)Difference" << endl << "3)Multiplication" << endl << "4)Division" << endl;
+    cin >> op;
+
+    if(op <1 || op >4){
+      cerr << "Wrong choice, please select a valid operation number [1-4]" << endl;
+    }
+
+    else if(op == 4 && b == 0){
+      cerr << "Can't divide a number for 0, please select another operation" << endl;
+    }
+
+    else{
+      break;
+    }
+  }
+
+  int result = switchOp(op, a, b);
+  
   cout << result << endl << endl;
```

Comment:
hai definito due variabili result, potresti non definirne neanche una in questo caso e fai 
```C++
cout << switchOp(op, a, b) << endl << endl;
```